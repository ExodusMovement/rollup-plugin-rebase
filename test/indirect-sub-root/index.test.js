import { bundle, clean, list, read } from "../util"

const root = __dirname

test("Indirect Sub Root", async () => {
  await bundle(root, "src/index.js", "output/index.js")

  expect(await list(root, "output")).toMatchSnapshot()
  expect(await read(root, "output/index.js")).toMatchSnapshot()

  await clean(root, "output")
})
