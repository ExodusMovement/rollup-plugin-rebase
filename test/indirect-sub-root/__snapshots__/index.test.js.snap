// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Indirect Sub Root 1`] = `
"bsdudxaF.md
index.js"
`;

exports[`Indirect Sub Root 2`] = `
"import content from './bsdudxaF.md';

function index() {
  return content
}

export default index;
"
`;
